src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -6,7 +6,8 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ClimbSubsystem extends SubsystemBase {
-    public ClimbSubsystem(WPI_TalonFX climbFixed1, WPI_TalonFX climbFixed2, WPI_TalonFX climbAngled1, WPI_TalonFX climbAngled2, DoubleSolenoid climbAngle) {
+    public ClimbSubsystem(WPI_TalonFX climbFixed1, WPI_TalonFX climbFixed2, WPI_TalonFX climbAngled1,
+            WPI_TalonFX climbAngled2, DoubleSolenoid climbAngle) {
 
     }
```

Comment:
Please undo the code reformatting to the other subsystems, as it'll cause painful merge conflicts for their PRs. They should do the formatting themselves as part of their PR.

Reply:
Thought I got rid of all of these...